oak/server/rust/oak_runtime/src/node/grpc/server/mod.rs

```diff
@@ -275,9 +333,12 @@ impl UnaryService<Vec<u8>> for GrpcRequestHandler {
 
     fn call(&mut self, request: tonic::Request<Vec<u8>>) -> Self::Future {
         let handler = self.clone();
+
+        let oak_label_result = get_oak_label(&self.metadata_map);
```

Comment:
Is this map being updated only once?
If not, since it's not inside a future, changes will not proceed

Reply:
Not sure what you mean by "will not proceed", but the map is updated only once from the initial request, all the metadata is available at that point. Is this fine, or am I missing something?